org.adempiere.base/src/org/compiere/db/partition/ITablePartitionService.java

```diff
@@ -99,4 +101,27 @@ public interface ITablePartitionService {
 	 * @param processInfo
 	 */
 	public void reattachPartition(MTable table, X_AD_TablePartition partition, String trxName, ProcessInfo processInfo);
+	
+	/**
+	 * Get default partition name for table
+	 * @param tableName
+	 * @return String default partition name for table
+	 */
+	public String getDefaultPartitionName(String tableName);
+	
+	/**
+	 * Parse table partition name
+	 * @param tableName
+	 * @param partitionCols
+	 * @return table partition name
+	 */
+	public String parsePartitionName(String tableName, Object[] partitionCols);
```

Comment:
what's the use case of this method ?

Reply:
E.g. I'm using the getPartitionName it in a process where I need to manually create/attach a table partition in the DB, based on the existing table definition in the application. The parsePartitionName method is used in getPartitionName